releasenotes/notes/49537.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+  - 49537
+releaseNotes:
+  - |
+    **Fixed** an issue that when using `withoutHeaders` matching,
```

Comment:
Can you update on what will it fix instead of documenting solution?

Reply:
Good suggestion, updated.